Sources/Player.swift

```diff
@@ -441,7 +458,7 @@ extension Player {
         let keys: [String] = [PlayerTracksKey, PlayerPlayableKey, PlayerDurationKey]
 
         self._asset.loadValuesAsynchronously(forKeys: keys, completionHandler: { () -> Void in
-            DispatchQueue.main.sync(execute: { () -> Void in
+            DispatchQueue.main.async(execute: { () -> Void in
```

Comment:
Given the new guard that makes sure the view is loaded before getting in here, I'm not sure if this is necessary. I'm still on version 0.2.0 and without this, I am getting an exception. I figured I would switch it to async as it seems like a better option unless it's needed to be synchronous. What are your thoughts?